middleware/http/sentinel/middleware.go

```diff
@@ -0,0 +1,165 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation and Dapr Contributors.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+package sentinel
+
+import (
+	"encoding/json"
+	"fmt"
+
+	sentinel "github.com/alibaba/sentinel-golang/api"
+	"github.com/alibaba/sentinel-golang/core/base"
+	"github.com/alibaba/sentinel-golang/core/config"
+	"github.com/dapr/components-contrib/middleware"
+	"github.com/dapr/dapr/pkg/logger"
```

Comment:
This has moved to `"github.com/dapr/kit/logger"`

Reply:
fixed.